app/pages/lab/mobile/mobile-section-container.jsx

```diff
@@ -24,6 +24,21 @@ function taskQuestionNotTooLong({ task }) {
   return convertBooleanToValidation(question ? question.length < VALID_QUESTION_LENGTH : false);
 }
 
+// A drawing task has task.instruction, instead of task.question, as the property edited in the "Main Text" input in the drawing task editor
+function taskInstructionNotTooLong({ task }) {
+  const { instruction } = task;
+  // TODO: Code to remove markdown images from character count. Change must be made in mobile app checks before addition.
+  // if (instruction) {
+  //   const matchArray = instruction.match(MARKDOWN_IMAGE);
+  //   if (matchArray) {
+  //     matchArray.map((imageLink) => {
+  //       instruction = instruction.replace(imageLink, '');
+  //     });
+  //   }
+  // }
```

Comment:
I was going to ask, "Why is there such a huge TODO here if you're not going to implement it?", but then I read the whole JSX code and saw that AH, that's inherited from `taskQuestionNotTooLong()`

Oh man, how long has that TODO been sitting there?

Reply:
Good question! I added reference to the related mobile issue in the comment for more context, easier tracking